[34m[1mwandb[39m[22m: [32m[41mERROR[39m[49m Attempted to change value of key "model_name" from distilbert-base-uncased to bert-base-uncased
[34m[1mwandb[39m[22m: [32m[41mERROR[39m[49m If you really want to do this, pass allow_val_change=True to config.update()
Traceback (most recent call last):
  File "C:\Users\bais_\source\repos\Progetto\NLP-customer-classification\src\scripts\train.py", line 23, in <module>
    wandb.config.update(config)
  File "C:\Users\bais_\anaconda3\Lib\site-packages\wandb\sdk\wandb_config.py", line 184, in update
    sanitized = self._update(d, allow_val_change)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bais_\anaconda3\Lib\site-packages\wandb\sdk\wandb_config.py", line 177, in _update
    sanitized = self._sanitize_dict(
                ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bais_\anaconda3\Lib\site-packages\wandb\sdk\wandb_config.py", line 264, in _sanitize_dict
    k, v = self._sanitize(k, v, allow_val_change)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bais_\anaconda3\Lib\site-packages\wandb\sdk\wandb_config.py", line 285, in _sanitize
    raise config_util.ConfigError(
wandb.sdk.lib.config_util.ConfigError: Attempted to change value of key "model_name" from distilbert-base-uncased to bert-base-uncased
If you really want to do this, pass allow_val_change=True to config.update()